CONTRIBUTORS.rst

```diff
@@ -85,6 +85,7 @@ Listed in alphabetical order.
   Daniel Hepper            `@dhepper`_                   @danielhepper
   Daniele Tricoli          `@eriol`_
   David Díaz               `@ddiazpinto`_                @DavidDiazPinto
+  Davit Tovmasyan          `@davitovmasyan`_
```

Comment:
The `@davitovmasyan`_ is a link in Sphinx, but it's currently a broken one. You also need a line at the bottom of this file:

```
.. _@davitovmasyan: https://github.com/davitovmasyan
```

Reply:
@browniebroke Fixed.